static/src/javascripts/bootstraps/enhanced/notifications.js

```diff
@@ -80,46 +75,12 @@ define([
 
             fastdom.write( function() {
                 $follow.html(src);
-                bean.one($follow[0], 'click', '.js-notifications__button', handler);
+                bean.one($follow[0], 'click', '.js-notifications__toggle', handler);
```

Comment:
I know this isn't new but it's worth checking `$follow` length before trying to access the array to avoid errors.